lib/MusicBrainz/Server/Entity/URL/ChangeTip.pm

```diff
@@ -0,0 +1,41 @@
+package MusicBrainz::Server::Entity::URL::ChangeTip;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name {
+    my $self = shift;
+
+    if ($self->url =~ m{^https?://(?:www.)?changetip.com/tipme/([^/?#]+)$}i) {
+        return $1 . '.tip.me';
+    }
+    else {
+        return "ChangeTip";
+    }
+};
+
+__PACKAGE__->meta->make_immutable;
+no Moose;
+1;
+
+=head1 COPYRIGHT
+
+Copyright (C) 2016 MetaBrainz Foundation
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
```

Comment:
The current address of the FSF is `51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA`.

However, it actually [recommends](https://www.gnu.org/licenses/gpl-howto.html) using the following text for the final paragraph nowadays:

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.